src/content/overview.md

```diff
@@ -107,15 +107,15 @@ class Point {
 }
 ```
 
-{{site.alert.info}}
-  This example is running in an embedded [DartPad](/tools/dartpad).
-  You can also
-  <a href="{{site.dartpad}}/bc63d212c3252e44058ff76f34ef5730"
-  target="_blank" rel="noopener">open this example in its own window</a>.
-{{site.alert.end}}
+:::note
+This example is running in an embedded [DartPad](/tools/dartpad).
+You can also
+<a href="{{site.dartpad}}/bc63d212c3252e44058ff76f34ef5730"
+target="_blank" rel="noopener">open this example in its own window</a>.
```

Comment:
FYI, external links like these aren't displayed with an external-link icon anymore.

Reply:
Yeah I left this out on purpose to figure out as follow up.

The Google documentation style guide recently [updated its guidelines](https://developers.google.com/style/whats-new#november-8,-2023) to suggest [not adding external link icons](https://developers.google.com/style/link-text#write-link-text:~:text=Don%27t%20use%20an%20external%20link%20icon%20to%20indicate%20that%20the%20link%20goes%20to%20a%20different%20domain%20or%20server) and instead improve surrounding context. They explained that "Readers and writers have expressed confusion about the meaning and usage of this icon".

My initial thinking is that in most cases it's sufficient to explain the destination in the surrounding text/context. However, in UI elements like the sidenav, I feel external icons might still necessary.

Reply:
> The Google documentation style guide recently [updated its guidelines](https://developers.google.com/style/whats-new#november-8,-2023) to suggest [not adding external link icons](https://developers.google.com/style/link-text#write-link-text:~:text=Don%27t%20use%20an%20external%20link%20icon%20to%20indicate%20that%20the%20link%20goes%20to%20a%20different%20domain%20or%20server) and instead improve surrounding context. They explained that "Readers and writers have expressed confusion about the meaning and usage of this icon".

Oh! That is very good to know. Thanks for providing those links to the style guide.